### PR TITLE
removed vouch display for non admins

### PR DESCRIPTION
**Pull request by kabobrocks**
just moved all vouching display to admins for now

### CODE REVIEW

File: src/templates/player/captains.pug [hunk 1]
```diff
@@ -41,8 +41,9 @@ block content
             if user && user.isAdmin
               th Will Captain?
               th Captain Approved
-            th Statement
-            th Vouched?
+            if user && user.isAdmin
+              th Statement
+              th Vouched?
```
Comment (dambrisco) [inline src/templates/player/captains.pug]: Not sure we should be hiding statement here. Also redundant `if` check.

File: src/templates/player/captains.pug [hunk 2]
```diff
@@ -75,13 +76,14 @@ block content
...
-                  span.icon.has-text-danger.has-icon
-                    i.far.fa-lg.fa-times-circle
+              if user && user.isAdmin
+                td
+                  if player.is_vouched || player.has_played
+                    span.icon.has-text-success.has-icon
+                      i.far.fa-lg.fa-check-circle
+                  else
+                    span.icon.has-text-danger.has-icon
+                      i.far.fa-lg.fa-times-circle
```
Comment (dambrisco) [inline src/templates/player/captains.pug]: Redundant `if`

File: src/templates/player/list.pug [hunk 3]
```diff
@@ -42,8 +42,9 @@ block content
               th Will Captain?
               th Captain Approved
             th Statement
-            th Vouched?
-            th Draftable?
+            if user && user.isAdmin
```
Comment (dambrisco) [inline src/templates/player/list.pug]: Redundant `if`

File: src/templates/player/list.pug [hunk 4]
```diff
@@ -76,20 +77,21 @@ block content
...
+                  else
+                    span.icon.has-text-danger.has-icon
+                      i.far.fa-lg.fa-times-circle
+                td
+                  if player.is_draftable
+                    span.icon.has-text-success.has-icon
+                      i.far.fa-lg.fa-check-circle
+                  else
+                    span.icon.has-text-danger.has-icon
+                      i.far.fa-lg.fa-times-circle
```
Comment (dambrisco) [inline src/templates/player/list.pug]: Redundant `if`

File: src/templates/player/standins.pug [hunk 5]
```diff
@@ -37,8 +37,9 @@ block content
             th Adjusted Rank
             th Dotabuff&nbsp;/ Opendota
             th Statement
-            th Vouched?
-            th Draftable?
+            if user && user.isAdmin
```
Comment (dambrisco) [inline src/templates/player/standins.pug]: Redundant `if`

File: src/templates/player/standins.pug [hunk 6]
```diff
@@ -61,20 +62,21 @@ block content
...
+                  else
+                    span.icon.has-text-danger.has-icon
+                      i.far.fa-lg.fa-times-circle
+                td
+                  if player.is_draftable
+                    span.icon.has-text-success.has-icon
+                      i.far.fa-lg.fa-check-circle
+                  else
+                    span.icon.has-text-danger.has-icon
+                      i.far.fa-lg.fa-times-circle
```
Comment (dambrisco) [inline src/templates/player/standins.pug]: Redundant `if`

**Comment by kabobrocks** [comment]
all changes fixed

**Pull request merged**